### PR TITLE
Adding my Medium article: Cypress + OneLogin

### PR DESCRIPTION
**Pull request by jedicwb**
Cypress + OneLogin: using API to authenticate before testing

### CODE REVIEW

File: source/_data/blogs.yml [hunk 1]
```diff
@@ -8,6 +8,14 @@ small:
 
 # large links with thumbnails, links, embedded video
 large:
+  - title: "Cypress + OneLogin: using API to authenticate before testing"
+    url: https://medium.com/@jedicwb/cypress-io-onelogin-using-api-to-authenticate-before-testing-a6e2412b356c
+    sourceName: Medium
+    sourceUrl: https://medium.com/
+    author: Jonathan Machado
+    date: August 24, 2020
+    img: https://miro.medium.com/max/576/1*-9hvi8ko4HWxF2jCA_JAQw.png
```
Comment (jennifer-shehane) [inline source/_data/blogs.yml:17]: The image should be a local image located within the repo.

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]
We need the images local to the repo, so that the image does not disappear on us.

**Comment by jedicwb** [comment]
Hi, Jennifer.

Here is the new pull request with the img added in:
https://github.com/cypress-io/cypress-documentation/pull/3107
Thanks for the heads up!

Em ter., 25 de ago. de 2020 às 03:41, Jennifer Shehane <
notifications@github.com> escreveu:

File: source/_data/blogs.yml (hunk 1 above)
Comment (jedicwb) [reply source/_data/blogs.yml:17]: I've added a new PR with the additional img:
https://github.com/cypress-io/cypress-documentation/pull/3107
Thank for the heads Up.

**Pull request closed without merging**